src/game/Strategic/Strategic_Movement.cc

```diff
@@ -2720,8 +2720,13 @@ void RetreatGroupToPreviousSector(GROUP& g)
 {
 	AssertMsg(!g.fBetweenSectors, "Can't retreat a group when between sectors!");
 
-	UINT8 direction = 255;
-	if (g.ubPrev.x != 16 || g.ubPrev.y != 16)
+	if (g.ubPrev.x == 16 && g.ubPrev.y == 16)
```

Comment:
Would be cleaner to use IsValid(), but that allows 16. I find the original check suspect — why exclude this one sector?

Reply:
The problem is trying to fit 256 different sector ids in one byte and still having a value to indicate 'invalid'. There is at least one other place, [in AutoResolve](https://github.com/ja2-stracciatella/ja2-stracciatella/blob/ff9cf59cdb0860c90e258ef43ac8a403ef058308/src/game/Strategic/Auto_Resolve.cc#L3889), which uses P16 as the odd one out.

Reply:
Ok. It's inaccessible anyway, likely in all modded versions as well.